nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -1062,7 +1065,7 @@ def predict_step(
             return_hypotheses=False,
         )[0]
 
-        return text
+        return (text, cuts)
```

Comment:
```suggestion
        return (text, batch.cuts)
```

and remove `cuts = batch.cuts` above.